Mathlib/Analysis/SpecialFunctions/Gamma/Deligne.lean

```diff
@@ -0,0 +1,196 @@
+/-
+Copyright (c) 2024 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+
+import Mathlib.Analysis.SpecialFunctions.Gamma.Beta
+
+/-!
+# Deligne's archimedean Gamma-factors
+
+In the theory of L-series one frequently encounters the following functions (of a complex variable
+`s`) introduced in Deligne's landmark paper *Valeurs de fonctions L et periodes d'integrales*:
+
+$$ \Gamma_{\mathbb{R}}(s) = \pi ^ {-s / 2} \Gamma (s / 2) $$
+
+and
+
+$$ \Gamma_{\mathbb{C}}(s) = 2 (2 \pi) ^ {-s} \Gamma (s). $$
+
+These are the factors that need to be included in the Dedekind zeta function of a number field
+for each real, resp. complex, infinite place.
+
+(Note that these are *not* the same as Mathlib's `Real.Gamma` vs. `Complex.Gamma`; Deligne's
+functions both take a complex variable as input.)
+
+This file defines these functions, and proves some elementary properties, including a reflection
+formula which is an important input in functional equations of (un-completed) Dirichlet L-functions.
+-/
+
+
+open Filter Topology Asymptotics Real Set MeasureTheory
+open Complex hiding abs_of_nonneg
+
+/-- Deligne's archimedean Gamma factor for a real infinite place, see
```

Comment:
Can you emphasize in the docstring that this is different from `Real.Gamma` and takes a complex variable as input? Could you also put the whole file in the `Complex` namespace to avoid cluttering the root namespace?